runelite-api/src/main/java/net/runelite/api/SpritePixels.java

```diff
@@ -53,4 +55,6 @@
 	void toBufferedImage(BufferedImage img);
 
 	void setOutline(int outlineColor);
+
+	BufferedImage toBufferedOutline(int color);
```

Comment:
I would probably do same thing as toBufferedImage does (e.g accept BufferedImage as param, and color as second param)

Reply:
Also, if you need only outline, merge the getOutlinePixels with toBufferedOutline